```diff
{% raw %}--- a/amplify/backend/function/S3Trigger93fd9546/src/package.json
+++ b/amplify/backend/function/S3Trigger93fd9546/src/package.json
@@ -3,5 +3,8 @@
   "version": "2.0.0",
   "description": "Lambda function generated by Amplify",
   "main": "index.js",
-  "license": "Apache-2.0"
+  "license": "Apache-2.0",
+  "dependencies": {
+    "jimp": "^0.8.4"
+  }
 }{% endraw %}
```